### PR TITLE
[Wazo 2909] - python3

### PR DESCRIPTION
**Pull request by sopelj**
Depends on: #49 

Testing note: Since the new plugin repo has not yet been published, if you want to test plugins, you must build them manually [from here](https://github.com/wazo-platform/wazo-provd-plugins/tree/WAZO-2910-python3-plugins)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -7,10 +7,10 @@ automat==0.6.0  # from twisted
...
-pyopenssl==19.0.0
+pyopenssl>=21.0.0,<23.0.0
+cryptography>=2.1.4 # for pyopenssl
 pyyaml==3.13
 requests==2.21.0
-six==1.12.0
 stevedore==1.29.0
 twisted==18.9.0
-zope.interface==4.3.2
+zope.interface==4.7.2
```
Comment (bloom1) [inline requirements.txt:16]: I thought you had replaced the zope interfaces by ABCMeta?

File: requirements.txt [hunk 2]
```diff
@@ -7,10 +7,10 @@ automat==0.6.0  # from twisted
 incremental==16.10.1  # from twisted
 jinja2==2.10
 markupsafe==1.1.0  # from jinja2
-pyopenssl==19.0.0
+pyopenssl>=21.0.0,<23.0.0
```
Comment (bloom1) [inline requirements.txt:10]: In debian 10 there is python3-openssl in version 19.0.0 - do you know if it is really necessary to use >= 21.0.0?

File: requirements.txt (hunk 1 above)
Comment (sopelj) [reply requirements.txt:16]: I did in all our classes. However unfortunately Twisted still uses it....

File: requirements.txt (hunk 2 above)
Comment (bloom1) [reply requirements.txt:10]: Ok: unit tests fail with pyopenssl 19.0.0 from pypi, but running the tests locally on the stack with the debian-provided 19.0.0 works. They probably patched it.

**Review by bloom1 — APPROVED** [review]
Excellent work :grin: You even took the time to fix spelling errors and fix some variable shadowing. You left the place cleaner than when you came in. Thank you!

**Pull request merged**